content/packit-as-a-service.md

```diff
@@ -4,31 +4,33 @@ date: 2019-06-22
 draft: false
 ---
 
-Packit-as-a-Service is a GitHub App (https://developer.github.com/apps/about-apps/).
+Packit-as-a-Service is a [GitHub App](https://developer.github.com/apps/about-apps/)
+that helps you continuously ensure that your project work in Fedora OS.
 
 ### Easy to use
 
-Using Packit Service is very straightforward: add one config file to your repository
-([see .packit.yaml format](https://github.com/packit-service/packit/blob/master/docs/configuration.md)),
-followed by an RPM spec file and you're good to go. Besides the service, packit is also a tool
-([see packit README.md](https://github.com/packit-service/packit/blob/master/README.md)),
-so you can always try things locally on your own.
+Using Packit Service is very straightforward: add one [config
+file](https://github.com/packit-service/packit/blob/master/docs/configuration.md)
+to your repository together with the RPM spec file and you're good to go. 
 
 Packit Service validates your pull requests by building your software in Fedora OS.
-Once the builds are done, packit lets you know how to install the
+Once the builds are done, Packit lets you know how to install the
 RPMs inside your environment so you can try the changes before merging them.
 
+Packit is also available as a [CLI tool](https://github.com/packit-service/packit/blob/master/README.md),
+so you can always try things locally on your own.
+
 <!--more-->
 
-## Steps how to integrate Packit-as-a-Service into your projects or organizations
+## Integrating Packit-as-a-Service into your project or organization
 
 There are two ways to install Packit-as-a-Service GitHub app into your repository.
 
 ### Install from Packit-as-a-Service page
-1. From the [Packit-service page](https://github.com/apps/packit-as-a-service) page,
-in the left sidebar, click "Install App"
+1. From the [Packit-service page](https://github.com/apps/packit-as-a-service),
+in the left sidebar, click "Install App" (alternatively, click "Configure" in the right sidebar)
 2. Click "Install" next to the organization or user account containing the correct repository
-3. Install the "Packit-as-a-Service" on all repositories or to certain repositories
+3. Install "Packit-as-a-Service" to repositories where you wish to use the service
```

Comment:
I would use the original section. `on all repositories or to certain repository`.

Reply:
It sounded weird - it's obvious that you use it either on one, two or as many repos as you want.  Or is it not an expected option?